pyproject.toml

```diff
@@ -23,7 +24,7 @@ dependencies = [
 requires-python = ">=3.9"
 
 [project.urls]
-Home = "https://git.torchbox.com/internal/django-birdbath"
+Home = "https://github.com/torchbox/django-birdbath"
```

Comment:
suggestion: add Django >= 4.2 to the dependencies list.
This is, after all, django-birdbath ;)

Reply:
Hah, good point!